torch/testing/_internal/logging_utils.py

```diff
@@ -117,6 +117,9 @@ def setUpClass(cls):
         cls._exit_stack.enter_context(
             unittest.mock.patch("torch._dynamo.config.suppress_errors", True)
         )
+        cls._exit_stack.enter_context(
+            unittest.mock.patch("torch._dynamo.config.verbose", False)
+        )
```

Comment:
Can we fix whatever code is doing an unbracketed edit of verbose? This is just papering over the problem, if someone is unlucky to write a test that is also affected in this way you'll have the same problem